charts/riff/requirements.yaml

```diff
@@ -1,11 +1,29 @@
 dependencies:
   - name: cert-manager
     condition: cert-manager.enabled
```

Comment:
cert-manager should be installed if any of the runtime/build tags are defined

Reply:
hmm, maybe we could add all tags then for cert-manager dependency

Reply:
exactly